### PR TITLE
Upgraded to sympy 1.6

### PR DESCRIPTION
**Pull request by GarkGarcia**
This is a follow up to https://github.com/mathics/Mathics/issues/799.

### CODE REVIEW

**Comment by GarkGarcia** [comment]
The tests are passing locally, I'm not sure why the builds are failing.

File: setup.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
     INSTALL_REQUIRES += ['cython>=0.15.1']
 
 # General Requirements
-INSTALL_REQUIRES += ['sympy>=1.6.2', 'django >= 1.8, < 1.12',
+INSTALL_REQUIRES += ['sympy==1.6.2', 'django >= 1.8, < 1.12',
```
Comment (suhr) [inline setup.py:66]: That's a little bit too rigid version range.

Comment (GarkGarcia) [reply setup.py]: What about `sympy>=1.6`?

**Comment by GarkGarcia** [comment]
Apparently there's a bug in SymPy 1.6.2 which causes some of our tests to fail (see https://github.com/sympy/sympy/issues/20051)

**Comment by GarkGarcia** [comment]
> Apparently there's a bug in SymPy 1.6.2 which causes some of our tests to fail (see [sympy/sympy#20051](https://github.com/sympy/sympy/issues/20051))

As stated in https://github.com/sympy/sympy/issues/20051#issuecomment-687655927, this is not a bug at all.

**Comment by GarkGarcia** [comment]
There's a single test that is not passing. I've submitted a bug report to [sympy/sympy](https://github.com/sympy/sympy) (https://github.com/sympy/sympy/issues/20053) to gatter information on how we could address it.

**Comment by GarkGarcia** [comment]
@suhr @wolfv It's finally done 😅️ Can I merge the PR?

**Review by suhr — APPROVED** [review]

**Pull request merged**